package.json

```diff
@@ -67,6 +64,7 @@
     "onscroll"
   ],
   "dependencies": {
-    "consolidated-events": "^1.0.1"
+    "consolidated-events": "^1.0.1",
+    "prop-types": "^15.5.8"
```

Comment:
I think it was a mistake to specify such a new version here. I think I'll change to `^15.0.0` instead, so that we won't cause duplicate packages for people on older versions.